bin/ck8s

```diff
@@ -35,7 +35,9 @@ usage() {
   echo "  update-ips <wc|sc|both> <apply|dry-run>         Automatically fetches and applies the IPs for network policies" 1>&2
   echo "  upgrade <wc|sc|both> <vX.Y> apply               runs all apply steps upgrading the environment" 1>&2
   echo "  upgrade <wc|sc|both> <vX.Y> prepare             runs all prepare steps upgrading the configuration" 1>&2
+  echo "  unlock-upgrade <wc|sc|both>                     reset a migration to allow trying again" 1>&2
```

Comment:
Nit, could this be `upgrade-unlock` instead? Makes similar commands stay together.
Or perhaps `upgrade <wc|sc|both> unlock` to be more similar in structure to the rest of the upgrade commands.

Reply:
> Nit, could this be `upgrade-unlock` instead?

Sure

> Or perhaps `upgrade <wc|sc|both> unlock`

Finding a good way to bypass the existing path that requires a version number was a bit awkward.